fyrox-impl/src/scene/mesh/surface.rs

```diff
@@ -94,18 +95,45 @@ pub struct InputBlendShapeData {
     pub tangents: FxHashMap<u32, Vector3<f16>>,
 }
 
+#[repr(C)]
+#[derive(Default, Clone, Copy)]
+struct PodVector3 {
+    x: f16,
+    y: f16,
+    z: f16,
+}
+
+unsafe impl Pod for PodVector3 {}
+unsafe impl Zeroable for PodVector3 {}
+
+impl From<Vector3<f16>> for PodVector3 {
+    fn from(v: Vector3<f16>) -> Self {
+        PodVector3 {
+            x: v.x,
+            y: v.y,
+            z: v.z,
+        }
+    }
+}
+
+impl From<PodVector3> for Vector3<f16> {
+    fn from(pv: PodVector3) -> Self {
+        Vector3::new(pv.x, pv.y, pv.z)
+    }
+}
+
```

Comment:
`half` crate has `bytemuck` feature as well, so this `PodVector3` can be replaced by simply enabling the `bytemuck` feature.